app/src/main/res/layout/activity_user_profile.xml

```diff
@@ -116,6 +116,25 @@
                     android:text="Save profile" />
             </LinearLayout>
 
+            <Space
+                android:layout_width="match_parent"
+                android:layout_height="26dp" />
+
+            <LinearLayout
+                android:layout_width="match_parent"
+                android:layout_height="match_parent"
+                android:layout_weight="3"
+                android:orientation="horizontal">
+
+                <Button
+                    android:id="@+id/viewOwnListingsButton"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_weight="1"
+                    android:onClick="viewOwnListings"
+                    android:text="View my listings" />
+            </LinearLayout>
+
             <Space
                 android:layout_width="match_parent"
                 android:layout_height="46dp" />
```

Comment:
constant

Reply:
Layout is full of hardcoded dimensions, I guess we will go over it when re-doing the UI